libraries/AP_RangeFinder/AP_RangeFinder_BLPing.cpp

```diff
@@ -86,6 +86,7 @@ bool AP_RangeFinder_BLPing::get_reading(float &reading_m)
 bool AP_RangeFinder_BLPing::get_signal_quality_pct(int8_t &quality_pct) const
 {
     if (status() != RangeFinder::Status::Good) {
+        quality_pct = RangeFinder::SIGNAL_QUALITY_NODATA;
```

Comment:
Its a bit odd to return false for no value but also fill in the reference.

Reply:
I agree it is odd. My preference is to initialize output parameters to cut down on chances for uninitialized data floating around the system. Happy to remove this.

Reply:
The changed function signature (int8_t get_signal_quality()) avoids this issue.